worldedit-core/src/main/java/com/sk89q/worldedit/world/registry/BlockRegistry.java

```diff
@@ -94,7 +94,7 @@ default BlockMaterial getMaterial(BlockState state) {
     /**
      * Register all blocks.
      */
-    default Collection<String> values() {
```

Comment:
This breaks API compatibility

Reply:
Sorry for not noticing this, I rolled back the method name and changed the comment in latest changes